pkg/solana/txm/txm.go

```diff
@@ -748,14 +748,11 @@ func (txm *Txm) Enqueue(ctx context.Context, accountID string, tx *solanaGo.Tran
 	// Perform compute unit limit estimation after storing transaction
 	// If error found during simulation, transaction should be in storage to mark accordingly
 	if cfg.EstimateComputeUnitLimit {
-		computeUnitLimit, err := txm.EstimateComputeUnitLimit(ctx, tx, id)
+		computeUnitLimit, err := txm.EstimateComputeUnitLimit(ctx, tx, id, cfg.ComputeUnitLimit)
 		if err != nil {
 			return fmt.Errorf("transaction failed simulation: %w", err)
 		}
-		// If estimation returns 0 compute unit limit without error, fallback to original config
-		if computeUnitLimit != 0 {
-			cfg.ComputeUnitLimit = computeUnitLimit
-		}
+		cfg.ComputeUnitLimit = computeUnitLimit
```

Comment:
This cfg  var naming is so confusing.
I got confused this is the Txm.cfg because of same name.
Could we rename to txCfg?